### PR TITLE
Additional Spelling corrections

### PR DESCRIPTION
**Pull request by ArieHein**
## Description

Second wave of spelling that were not included in wave 1

## Reviews

- [ ] Technical review
- [x] Editorial review
- [ ] Product review

### CODE REVIEW

File: content/manuals/engine/release-notes/27.md [hunk 1]
```diff
@@ -350,15 +350,15 @@ For a full list of pull requests and changes in this release, refer to the relev
...
   - When a port mapping only applies to `routed` mode, only addresses `0.0.0.0` or `::` are allowed and a host port must not be given.
   - Note that published container ports, in `nat` or `routed` mode, are accessible from any remote address if routing is set up in the network, unless the Docker host's firewall has additional restrictions. For example: `docker network create --ipv6 -o com.docker.network.bridge.gateway_mode_ipv6=routed mynet`.
   - The option `com.docker.network.bridge.gateway_mode_ipv4=<nat|routed>` is also available, with the same behavior but for IPv4.
-- If firewalld is running on the host, Docker creates policy `docker-forwarding` to allow forwarding from any zone to the `docker` zone. This makes it possible to configure a bridge network with a routable IPv6 address, and no NAT or masquerading. [moby/moby#47745](https://github.com/moby/moby/pull/47745)
+- If firewalled is running on the host, Docker creates policy `docker-forwarding` to allow forwarding from any zone to the `docker` zone. This makes it possible to configure a bridge network with a routable IPv6 address, and no NAT or masquerading. [moby/moby#47745](https://github.com/moby/moby/pull/47745)
```
Comment (dvdksn) [inline content/manuals/engine/release-notes/27.md:361]: This one is actually referring to `firewalld`

**Review by dvdksn — COMMENTED** [review]
💪 thank you again! One tiny comment for this one.

File: content/manuals/engine/release-notes/27.md (hunk 1 above)
Comment (ArieHein) [reply content/manuals/engine/release-notes/27.md]: good catch. trying to follow the accept.txt in the vale folder but i missed that one.

**Review by dvdksn — APPROVED** [review]
🫡♥️

**Pull request merged**